Documentation/contributing/release/backports.rst

```diff
@@ -152,8 +152,20 @@ Creating the backports branch
 Creating the backport pull request
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The backport pull-request may be created via the GitHub web interface, or
-alternatively you can use CLI tools to achieve these steps.
+The backport pull-request may be created via CLI tools, or alternatively
+you can use the GitHub web interface to achieve these steps.
+
+Via command-line tools
+^^^^^^^^^^^^^^^^^^^^^^
+
+These steps require all of the tools described in the :ref:`backport_setup`
+section above. It pushes the git tree, creates the pull request and updates
+the labels for the PRs that are backported, based on the ``YYYY-MM-DD.txt``
+file in the current directory.
+
+   .. code-block:: bash
+
+      # contrib/backporting/start-backport
```

Comment:
Should this be `submit-backport` instead?

Reply:
Brain fart, thanks for pointing it out :) Fixed now.